src/js/components/settings.js

```diff
@@ -0,0 +1,90 @@
+const timerSwitch = document.querySelector('.switch');
+const settingsSaveButton = document.querySelector('.settings__save');
+const questionsTimer = document.querySelector('.questions__timer__input');
+const volumeProgress = document.querySelector('.volume__progress');
+const questionsChoices = document.querySelectorAll('.questions__choices');
+const cardTimerIntervals = document.querySelectorAll('.card__timer__intervals');
+const pictureInfo = document.querySelector('.picture-info');
+const congratulationBg = document.querySelector('.congratulation__bg');
+const pictureInfoIndicator = document.querySelector('.picture-info__indicator');
+const pictureInfoButton = document.querySelector('.picture-info__button');
+const questionsTimerCurrent = document.querySelector(
+  '.questions__timer__current'
+);
+const artistsQuestionsSection = document.querySelector('.artists__questions');
+const pictureQuestionsSection = document.querySelector('.picture__questions');
+const questionsTimerTotal = document.querySelector('.questions__timer__total');
+const questionsAudioCorrect = document.querySelector(
+  '.questions__audio-correct'
+);
+const questionsAudioWrong = document.querySelector('.questions__audio-wrong');
+function setLocalStorage() {
+  let timerChecked = isChecked();
+  localStorage.setItem('volume', volumeProgress.value);
+  localStorage.setItem('timer', timerChecked);
+}
+settingsSaveButton.addEventListener('click', setLocalStorage);
+
+function getLocalStorage() {
+  timerSwitch.checked = localStorage.getItem('timer');
+  volumeProgress.value = localStorage.getItem('volume');
+}
+window.addEventListener('load', () => {
+  getLocalStorage();
+  if (
+    !artistsQuestionsSection.classList.contains('hide') ||
+    !pictureQuestionsSection.classList.contains('hide')
+  ) {
+    setTimer(isChecked());
+  }
+});
+
+export const isChecked = () => {
+  if (timerSwitch.checked) return true;
+  else return '';
+};
+volumeProgress.addEventListener('change', () => {
+  questionsAudioCorrect.volume = volumeProgress.value;
+  questionsAudioWrong.volume = volumeProgress.value;
+});
+export const setTimer = (checkbox) => {
+  if (checkbox) {
```

Comment:
It is better to write 
if (!checkbox) return;
and then normal code flow without wrapping in 'if'